source/js/plugins/Pixel.js/source/export.js

```diff
@@ -16,60 +16,112 @@ export class Export
         this.zoomLevel = zoomLevel;
         this.matrix = null;
         this.uiManager = uiManager;
+        this.layersCount = 3;
```

Comment:
Using magic numbers for the layers work only if the number of imported/exported layers is fixed. We should start thinking about how to make these as variables.